scripts/install.js

```diff
@@ -1,166 +1,36 @@
-  // http request.  the '...url.parse(ANGLE_BINARY_URI)' part fills in the host,
-  // path, protocol, etc from the ANGLE_BINARY_URI and then we set the agent to
-  // the default agent which is overridden a few lines down if there is a proxy
-  const options = {
-    ...url.parse(ANGLE_BINARY_URI),
-    agent: https.globalAgent,
-    headers: {'Cache-Control': 'no-cache'}
-  };
-
-  if (proxy !== '') {
-    options.agent = new HttpsProxyAgent(proxy);
-  }
-
-  const request = https.get(options, response => {
-    const bar = new ProgressBar('[:bar] :rate/bps :percent :etas', {
-      complete: '=',
-      incomplete: ' ',
-      width: 30,
-      total: parseInt(response.headers['content-length'], 10)
-    });
-
-    if (platform === 'win32') {
-      // Save zip file to disk, extract, and delete the downloaded zip file.
-      const tempFileName = path.join(__dirname, '_tmp.zip');
-      const outputFile = fs.createWriteStream(tempFileName);
-
-      response.on('data', chunk => bar.tick(chunk.length))
-          .pipe(outputFile)
-          .on('close', async () => {
-            const zipFile = new zip(tempFileName);
-            zipFile.extractAllTo(depsPath, true /* overwrite */);
-
-            await unlink(tempFileName);
-
-            // The .lib files for the two .dll files we care about have a name
-            // the compiler doesn't like - rename them:
-            await rename(
-                path.join(
-                    depsPath, 'angle', 'out', 'Release', 'libGLESv2.dll.lib'),
-                path.join(
-                    depsPath, 'angle', 'out', 'Release', 'libGLESv2.lib'));
-            await rename(
-                path.join(
-                    depsPath, 'angle', 'out', 'Release', 'libEGL.dll.lib'),
-                path.join(depsPath, 'angle', 'out', 'Release', 'libEGL.lib'));
-
-            if (callback !== undefined) {
-              callback();
-            }
-          });
-    } else {
-      // All other platforms use a tarball:
-      response
-          .on('data',
-              (chunk) => {
-                bar.tick(chunk.length);
-              })
-          .pipe(tar.x({C: depsPath, strict: true}))
-          .on('close', () => {
-            if (callback !== undefined) {
-              callback();
-            }
-          });
+const path = require("path");
+const fs = require("fs");
+const request = require("request");
+const fileName = require("./file-name");
+
+if (fs.existsSync(fileName.qualifiedName)) { return; }
+if (process.env["NODE_GLES_SKIP_BINARY_DOWNLOAD_FOR_CI"] || process.env["npm_config_node_gles_skip_binary_download_for_ci"]) { return; }
+
+// Get the version of the library;
+const pkg = require(path.resolve(__dirname, "..", "package.json"));
+const packageVersion = pkg.version;
+const baseUrl = process.env["NODE_GLES_BINARY_URL"] || process.env["npm_config_node_gles_binary_url"] || "https://github.com/Prior99/node-gles/releases/download";
```

Comment:
Url in this line needs to be adjusted to this repository.

Reply:
Are you hosting angle builds here? I wish I had more time to manage these, but I'm pretty busy with other projects and don't have the time to manage these builds.

Do you mind adding some documentation where these need to be uploaded? If it's easy for them to be uploaded maybe this is something that can be managed here.